docs/how-tos/testing-validation-tools.md

```diff
@@ -0,0 +1,175 @@
+# The shows the example of testing NeXus files with validation methods
```

Comment:
Did you want to keep this file? If not, it should be removed from the PR (we can keep it somewhere else) and the reference in validate_nexus-file.md should be removed.

Reply:
keep it.